bylaws.md

```diff
@@ -613,6 +619,18 @@ has a significant consulting relationship with an entity; or that
 person owns at least 1% of the equity or debt, or derivatives 
 thereof, of an entity.
 
+**Section 5.15. Limits on Co-affiliation of Board Members.** No more
+than one quarter (1/4) of the members of the Board of Directors may
+share a common affiliation as defined in Section 5.14. If the number
+of co-affiliated directors goes above the limit due to a change in
+employment or a corporate acquisition, then the longest-serving member(s)
+of the Board of Directors sharing that affiliation must resign before
+the next election of at-large members to bring the total number of 
+co-affiliated directors below the limit.
+
+A person who would bring the Board of Directors above the limit on
+co-affiliation is ineligible to be seated or appointed.
+
```

Comment:
I'm reading this to be in conflict with the above. 

If the number exceeds, the longest serving member should resign. If an added member would exceed, they are ineligible. 

Should this part be clarified with "at election"? Then I think it would make sense.

Reply:
These take place at different times. If there is a change in affiliation (for whatever reason) that takes the board above the limit, that needs to be resolved by the next meeting of the board of directors.

At any time (both at election time and between elections), if a particular board candidate would bring the number of co-affiliates above the limit, then that person is disqualified. It applies both at election (to be seated) and in between elections (to be appointed).